packages/openapi3/src/openapi.ts

```diff
@@ -1149,16 +1149,24 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
   ) {
     ph.name = parameter.name;
     ph.in = parameter.type;
-    if (parameter.type === "query") {
-      if (parameter.format === "csv") {
+    if (parameter.type === "query" || parameter.type === "header") {
```

Comment:
I think we need an extra check to ensure 'form' is not used with header parameters

Reply:
Added. I made it a warning with no default.